src/pages/Layout/Layout.js

```diff
@@ -1,20 +1,19 @@
 import { Outlet } from "react-router-dom";
 import { Link } from "./Link/Link";
+import './Link/Link.scss'
 
 
 
 const Layout = () => {
   return (
     <>
-      <nav>
-        <ul>
-          <li>
-            <Link to="/">Хто ми</Link>
-          </li>
-          <li>
-            <Link to="/app">Blogs</Link>
-          </li>
-        </ul>
+      <nav  className="naw-wrapper">
+        <div className="link-wrapper">
+          <Link to="/">Хто ми</Link>
+        </div>
+        <div>
```

Comment:
You can render this `<div>` inside of the `<Link>` component, so you don't have to wrap each `<Link>` in a `<div>`.

Reply:
this structure for true changing position link